### PR TITLE
[Makefile, tests] Windows support + Azure CI

### PR DESCRIPTION
**Pull request by ayushkovskiy**

### CODE REVIEW

**Comment by ayushkovskiy** [comment]
Green! :tada:

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -58,22 +58,23 @@ TRAIN_CMD?=python -u $(CODE_DIR)/train.py --data $(DATA_DIR)
...
 LOCAL_PORT?=2211
 
+
 ##### SECRETS ######
 
 # Google Cloud integration settings:
 GCP_SECRET_FILE?=neuro-job-key.json
 GCP_SECRET_PATH_LOCAL=${CONFIG_DIR}/${GCP_SECRET_FILE}
-GCP_SECRET_PATH_ENV=${PROJECT_PATH_ENV}/${GCP_SECRET_PATH_LOCAL}
+GCP_SECRET_PATH_ENV=$(PROJECT_PATH_ENV)/${GCP_SECRET_PATH_LOCAL}
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/Makefile]: minor, not related to Windows

File: {{cookiecutter.project_slug}}/Makefile [hunk 2]
```diff
@@ -145,8 +146,8 @@ setup: ### Setup remote environment
 		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):ro \
 		$(BASE_ENV_NAME) \
 		'sleep infinity'
-	$(NEURO) exec --no-key-check $(SETUP_JOB) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT) update && cat $(PROJECT_PATH_ENV)/apt.txt | xargs -I % $(APT) install --no-install-recommends % && $(APT) clean && $(APT) autoremove && rm -rf /var/lib/apt/lists/*'"
-	$(NEURO) exec --no-key-check $(SETUP_JOB) "bash -c '$(PIP) -r $(PROJECT_PATH_ENV)/requirements.txt'"
+	$(NEURO) exec --no-key-check -T $(SETUP_JOB) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT) update && cat $(PROJECT_PATH_ENV)/apt.txt | tr -d \"\r\\r\" | xargs -I % $(APT) install --no-install-recommends % && $(APT) clean && $(APT) autoremove && rm -rf /var/lib/apt/lists/*'"
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/Makefile]: - `-T` is a bug
- `tr -d` is required for Windows

File: {{cookiecutter.project_slug}}/Makefile [hunk 3]
```diff
@@ -341,7 +342,7 @@ hypertrain: _check_setup wandb-check-auth   ### Run jobs in parallel for hyperpa
 	@[ -f "$(WANDB_SWEEP_CONFIG_PATH)" ] \
 		&& echo "Using W&B sweep file: ./$(WANDB_SWEEP_CONFIG_PATH)" \
 		|| { echo "ERROR: W&B sweep config file not found: '$${PWD}/$(WANDB_SWEEP_CONFIG_PATH)'" >&2; false; }
-	wandb sweep $(WANDB_SWEEP_CONFIG_PATH) 2>&1 | tee /dev/stderr | grep 'Created sweep with ID: ' | awk 'NF{ print $$NF }' >> $(WANDB_SWEEPS_FILE)
+	wandb sweep $(WANDB_SWEEP_CONFIG_PATH) 2>&1 | tee /proc/self/fd/2 | grep 'Created sweep with ID: ' | awk 'NF{ print $$NF }' >> $(WANDB_SWEEPS_FILE)
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/Makefile]: interesting: `/dev/stderr` does not exist on Anaconda Prompt, whereas `/proc/self/fd/2` does!

File: h.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+echo 123 | grep kk
```
Comment (mariyadavydova) [inline h.sh]: What is this file for?

File: tests/e2e/cleanup.sh [hunk 5]
```diff
@@ -8,7 +8,7 @@ STORAGE_FILE="$OUTPUT_DIR/cleanup_storage.txt"
 echo "Cleaning up jobs..."
 
 echo "Reading jobs file: $JOBS_FILE"
-jobs=$([ -f $JOBS_FILE ] && cat $JOBS_FILE || true)
+jobs=$([ -f $JOBS_FILE ] && cat $JOBS_FILE | tr -d "\r\\r" || true)
```
Comment (ayushkovskiy) [inline tests/e2e/cleanup.sh]: maybe just `"\\r"` is enough?

File: h.sh (hunk 4 above)
Comment (ayushkovskiy) [reply h.sh]: oops

**Pull request merged**